diagnostic.md

```diff
@@ -57,13 +67,15 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('li').hide('li')
 <!-- your answer ends here -->
```

Comment:
```
$('li').parent().parent().hide();
```

read carefully. "Please write a method that traverses from a list item (`li`) DOM node _to the
  body DOM node, then hides it._"